Directory.Build.props

```diff
@@ -5,6 +5,7 @@
             Defines the lowest supported target framework for the extension.
             Used by server download / integration tests to ensure they run when only this SDK is installed.
         -->
+        <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
Noticed project loading did not like running on a .NET 10 preview while building net6.0 projects.